src/igvfd/schemas/tabular_file.json

```diff
@@ -114,6 +114,7 @@
                 "barcode to sample mapping",
                 "barcode to variant mapping",
                 "barcode to TF overexpression mapping",
+                "bin paired count",
```

Comment:
This enum also needs to be updated on the changelog for `Analysis Steps`.

Reply:
Added to the analysis step changelog.